tests/e2e/features/transactions/transfer.feature

```diff
@@ -0,0 +1,26 @@
+@transfer @regression @transactions 
+Feature: Transfer
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id1231
+  Scenario: Make a transfer in ETH
+    Given I am on the Main page
+    When I go to page "/transaction/zksync/era/send?network=era-goerli&address=0x9CC8DC9c4d73fC5647A4eE78A2e8EC49D447AeB8"
+    When I choose "ETH" as token and insert "0.0000000001" as amount
+    When I "confirm" transaction after clicking "Send to zkSync Era Testnet" button
+    Then Message "Transaction submitted" should be visible
+    Then Message "Transaction completed" should be visible
+    Then Element with "text" "Send" should be "visible"
+    Then Arrow element for "Transfer" external link should be "visible"
+    Then Arrow element for "Transfer" external link should be "clickable"
+
+  @id1276
```

Comment:
Steps themselves are good. please align steps in Allure at least, otherwise something looks missing https://allure.matterlabs.dev/project/17/test-cases/1276?treeId=30

Reply:
https://allure.matterlabs.dev/project/17/test-cases/1321?treeId=30 - Test scenario fixed